### ISSUE TITLE
SVG Images in Markdown not shown

### ISSUE THREAD

**Issue by M1S2** [whistyun/MdXaml#37]
In most of my projects I'm using some badges (from shields.io) at the top of my Readme.md file. E.g.:
`
[![Nuget Version](https://img.shields.io/nuget/v/AssemblyInfoHelper.svg)](https://www.nuget.org/packages/AssemblyInfoHelper/)
`

These badges result in SVG images that aren't correctly shown when the Markdown file is loaded with the MarkdownScrollViewer.
The resulting text is: 
```
!https://img.shields.io/nuget/v/AssemblyInfoHelper.svg
NotSupportedException:No imaging component suitable to complete this operation was found
```
Is it possible to support SVG images in markdown?
To show the Readme files (including the badges) I'm using a selfwritten library: https://github.com/M1S2/AssemblyInfoHelper
There is also a complete example Readme.md in this repository.

**Comment by whistyun** [comment]
This issue was addressed with #38
https://github.com/whistyun/MdXaml/issues/38#issuecomment-1264658411

**Comment by M1S2** [comment]
Thank you for this feature. It's working great.
